src/cli.js

```diff
@@ -275,6 +276,11 @@ Options
                 }
                 runner.testServer = value;
                 break;
+            
+            case "recursive":
+                noValue();
+                runner.isRecursive = true;
```

Comment:
If you want to put this value into the `Runner` object, please add it to the constructor in `runner.js` with a description (mostly for documentation purposes). I actually recommend making `isRecursive` a plain variable (declared under, say, the "Globals" section in `cli.js`, since the `Runner` object never actually deals with it.